src/bsp/platform/esp32/xi_bsp_sft_notifications_esp32.c

```diff
@@ -0,0 +1,8 @@
+/* Copyright (c) 2003-2017, LogMeIn, Inc. All rights reserved.
+ *
+ * This is part of the Xively C Client library,
+ * it is licensed under the BSD 3-Clause license.
+ */
```

Comment:
I'd rename this file to `xi_bsp_fwu_notifications_esp32.c`. Why? Because the FWU is already used as a BSP module. Even do this when this FWU notification is not only used from the FWU BSP but also from the IO FS BSP.